consensus/polybft/contractsapi/helper.go

```diff
@@ -0,0 +1,44 @@
+package contractsapi
+
+import (
+	"math/big"
+
+	"github.com/0xPolygon/polygon-edge/types"
+	"github.com/umbracle/ethgo/abi"
+)
+
+// StateTransactionInput is an abstraction for different state transaction inputs
+type StateTransactionInput interface {
+	// EncodeAbi contains logic for encoding arbitrary data into ABI format
+	EncodeAbi() ([]byte, error)
+	// DecodeAbi contains logic for decoding given ABI data
+	DecodeAbi(b []byte) error
+	// Type returns type of state transaction input
+	Type() StateTransactionType
```

Comment:
Maybe we can even remove `Type` function. It seems it is not used anywhere.

```suggestion
```

Reply:
Fix: https://github.com/0xPolygon/polygon-edge/pull/1166/commits/0761e2dc8fcade2f8f976bb1976629310e14bacf